shared/build.gradle.kts

```
@@ -28,21 +32,46 @@ kotlin {
         framework {
             baseName = "shared"
         }
-        xcodeConfigurationToNativeBuildType["Debug Staging"] = NativeBuildType.DEBUG
-        xcodeConfigurationToNativeBuildType["Debug Production"] = NativeBuildType.DEBUG
-        xcodeConfigurationToNativeBuildType["Release Staging"] = NativeBuildType.RELEASE
-        xcodeConfigurationToNativeBuildType["Release Production"] = NativeBuildType.RELEASE
+        xcodeConfigurationToNativeBuildType[XcodeConfiguration.DEBUG_STAGING] =
+            NativeBuildType.DEBUG
+        xcodeConfigurationToNativeBuildType[XcodeConfiguration.DEBUG_PRODUCTION] =
+            NativeBuildType.DEBUG
+        xcodeConfigurationToNativeBuildType[XcodeConfiguration.RELEASE_STAGING] =
+            NativeBuildType.RELEASE
+        xcodeConfigurationToNativeBuildType[XcodeConfiguration.RELEASE_PRODUCTION] =
+            NativeBuildType.RELEASE
     }
     
     sourceSets {
-        val commonMain by getting
+        val commonMain by getting {
+            dependencies {
+                implementation(Dependency.COROUTINES_CORE)
+
+                implementation(Dependency.KTOR_CORE)
+                implementation(Dependency.KTOR_SERIALIZATION)
+                implementation(Dependency.KTOR_LOGGING)
+                implementation(Dependency.KTOR_CIO)
+                implementation(Dependency.KTOR_CONTENT_NEGOTIATION)
+                implementation(Dependency.KTOR_JSON)
+                implementation(Dependency.KTOR_AUTH)
+
+                implementation(Dependency.NAPIER)
+
+                // settings
+                implementation(Dependency.SETTINGS)
+                implementation(Dependency.SETTINGS_SERIALIZATION)
+
+                //Json API
+                implementation(project(Module.JSONAPI_CORE))
```

And then updating this and syncing the project should work.
```suggestion
                implementation(Dependency.JSONAPI_CORE)
```